Doit/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
We already have a .gitignore file in the root folder. Please add your updates there.